tests/dummy/config/environment.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (environment) {
     podModulePrefix: 'dummy/pods',
     environment: environment,
     baseURL: '/',
-    locationType: 'hash',
+    locationType: 'auto',
```

Comment:
Forgot about this. Was playing with different browser behaviors in terms of states. It seems they handle popstate / hashchange a bit differently. Not really relevant, I'll revert this bit.